CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
+
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement that you might like to make. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
+
+If you would like to work on a fix for any issue, please assign the issue to yourself prior to creating a patch.
```

Comment:
I believe most people don't have the necessary rights to assign themselves to an issue